src/processTranslation.tsx

```diff
@@ -49,14 +49,11 @@ export const processTranslation = ({
     throw new Error(`Missing translation: "${key}"`);
   }
 
-  let suffix;
-
-  if (args.context) {
-    suffix = args.context;
+  let suffix = args.context;
 
   // @TODO: Update to the new format https://www.i18next.com/translation-function/plurals
```

Comment:
Is this TODO still relvant?

Reply:
Yes, I think:
We have 
```
single: `key`
plural: `key_plural`
```
But the i18next format is:
```
keys

{
  "key_one": "item",
  "key_other": "items",
  "keyWithCount_one": "{{count}} item",
  "keyWithCount_other": "{{count}} items"
}
sample

i18next.t('key', {count: 0}); // -> "items"
i18next.t('key', {count: 1}); // -> "item"
i18next.t('key', {count: 5}); // -> "items"
i18next.t('key', {count: 100}); // -> "items"
i18next.t('keyWithCount', {count: 0}); // -> "0 items"
i18next.t('keyWithCount', {count: 1}); // -> "1 item"
i18next.t('keyWithCount', {count: 5}); // -> "5 items"
i18next.t('keyWithCount', {count: 100}); // -> "100 items"
```